### ISSUE TITLE
Feature Request: Search function for website

### ISSUE THREAD

**Issue by mathiasconradt** [nextcloud/nextcloud.com#705]
There should be a fulltext-search available for the website.

**Comment by jospoortvliet** [comment]
Right... Wordpress has that feature, but how on earth that would work, esp considering we put all content in the theme - no clue. Sorry ;-)
